lingetic-nextjs-frontend/app/page.tsx

```
@@ -1,10 +1,317 @@
+            Our Philosophy
+          </h2>
+          <p className="text-lg mb-6 text-[#374151]">
+            Lingetic is built on the belief that the best way to learn a
+            language is through interaction with native speakers. This app
+            doesn't aim to replace these crucial real-world interactions.
+          </p>
+          <p className="text-lg mb-8 text-[#374151]">
+            Instead, it prepares you to make the most of them by equipping you
+            with the confidence and skills to get started.
+          </p>
+          <button className="bg-[#2563eb] hover:bg-blue-700 text-[#ffffff]">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)